src/main/asciidoc/streaming/jdbc/http-mysql/pcf.adoc

```diff
@@ -39,13 +39,13 @@ dataflow:>app import --uri {app-import-rabbit-maven}
 dataflow:>stream create --name mysqlstream --definition "http | jdbc --tableName=names --columns=name"
 Created new stream 'mysqlstream'
 
-dataflow:>stream deploy --name mysqlstream --properties "app.jdbc.spring.cloud.deployer.cloudfoundry.services=mysql"
+dataflow:>stream deploy --name mysqlstream --properties "deployer.jdbc.cloudfoundry.services=mysql"
```

Comment:
Isn't this still a valid application property and hence it needs to have `app.` as the prefix? Same is the case for other places as well?

Reply:
Nope, it doesn't work on the way it was written in the docs. And according to the latest docs on the cf deployer. `deployer.<app>.cloudfoundry.services` is the correct way.